chainstory/linkapp/consumers.py

```diff
@@ -0,0 +1,17 @@
+from channels.generic.websocket import AsyncWebsocketConsumer
+import json
+
+class MyConsumer(AsyncWebsocketConsumer):
+    async def connect(self):
+        await self.accept()
+
+    async def disconnect(self, close_code):
+        pass
+
+    async def receive(self, text_data):
```

Comment:
So with this PR, it's just receiving, not yet broadcasting? Or does it already do both?

Reply:
I'm actually testing that right now, it is less clear than I had originally thought